src/js/Content/Features/Community/MarketHome/FMarketLowestPrice.js

```diff
@@ -75,61 +57,72 @@ export default class FMarketLowestPrice extends Feature {
             const linkNode = node.querySelector(".market_listing_item_name_link");
             if (!linkNode) { continue; }
 
-            const m = linkNode.pathname.match(/\/(\d+)\/(.+)$/);
-            if (!m) { continue; }
-            const marketHashName = m[2];
+            const [, appid, marketHashName] = linkNode.pathname.match(/^\/market\/listings\/(\d+)\/([^/]+)/) || [];
+            if (!appid || !marketHashName) { continue; }
 
-            let priceData;
+            const lowestNode = node.querySelector(".es_market_listing_lowest");
+            const data = this._loadedMarketPrices[marketHashName] || await this._getPriceOverview(node, Number(appid), marketHashName);
 
-            if (this._loadedMarketPrices[marketHashName]) {
-                priceData = this._loadedMarketPrices[marketHashName];
-            } else {
-                priceData = await this._getPriceOverview(node, parseInt(m[1]), marketHashName);
+            if (!data) {
+                lowestNode.textContent = Localization.str.theworderror;
+                continue;
+            }
+
+            if (!("lowest_price" in data)) {
+                lowestNode.textContent = Localization.str.no_data;
+                continue;
             }
 
-            if (priceData !== null) {
-                this._insertPrice(node, priceData);
+            lowestNode.textContent = data.lowest_price;
+
+            const myPrice = Price.parseFromString(node.querySelector(".market_listing_price span span").textContent);
+            const lowPrice = Price.parseFromString(data.lowest_price);
+
+            if (myPrice.value <= lowPrice.value) {
+                lowestNode.classList.add("es_priced_lower"); // Our price matches the lowest price
+            } else {
+                lowestNode.classList.add("es_priced_higher"); // Our price is higher than the lowest price
             }
         }
     }
 
     async _getPriceOverview(node, appid, marketHashName) {
 
-        let done = false;
-        let priceData = null;
+        const country = User.storeCountry;
+        const currencyNumber = CurrencyManager.currencyTypeToNumber(CurrencyManager.storeCurrency);
+
+        let data;
+        let attempt = 0;
 
         do {
             try {
-                const data = await RequestData.getJson(
-                    `https://steamcommunity.com/market/priceoverview/?country=${User.storeCountry}`
-                    + `&currency=${CurrencyManager.currencyTypeToNumber(CurrencyManager.storeCurrency)}`
-                    + `&appid=${appid}`
-                    + `&market_hash_name=${marketHashName}`
+                data = await RequestData.getJson(
+                    `https://steamcommunity.com/market/priceoverview/?country=${country}&currency=${currencyNumber}&appid=${appid}&market_hash_name=${marketHashName}`
```

Comment:
This endpoint is used in several features, and is probably worth factoring out the avoid importing `User` and `CurrencyManager` just to build this URL.